charts/generic-service/templates/service.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 spec:
   ports:
     - name: {{ .Values.service.portName }}
+      protocol: {{ .Values.service.protocol }}
```

Comment:
Still unsure on if we would ever require `UDP`?

Reply:
Maybe, but it's been 4 years and we dont, this is mostly to match up with the NPs